main/res/values/strings.xml

```diff
@@ -596,6 +596,8 @@
     <string name="init_showwaypoint_description">If less than the given amount of caches are displayed on the map, their waypoints are shown additionally.</string>
     <string name="init_brouterThreshold">Calculate route</string>
     <string name="init_brouterThreshold_description">Maximum distance up to which brouter helper app (if installed) will calculate a route.</string>
+    <string name="init_distanceBeep">Distance beep</string>
```

Comment:
I suggest not to use the word beep.
What about "Distance signaling"?

Reply:
texts for settings title and info texts adjusted